listings/applications/erc721/src/tests.cairo

```diff
@@ -0,0 +1,90 @@
+#[cfg(test)]
+mod tests {
+    use super::*;
```

Comment:
The tests are written in Rust, not in Cairo. 

Once you rewrite them, you should run `bash scripts/cairo_programs_verifier.sh` from the root directory - it will pinpoint any potential issues in your code and verify all of the tests pass (see project's [Contributing guidelines](https://github.com/NethermindEth/StarknetByExample/blob/main/CONTRIBUTING.md#verification-script)).**

Reply:
That is because you renamed the file from _tests.cairo_ to _test_ERC721.cairo_, but didn't declare the corresponding module in _lib.cairo_.
Revert the file name to _tests.cairo_.